force-app/main/default/lwc/gePaymentGatewayManagement/gePaymentGatewayManagement.html

```diff
@@ -68,6 +68,17 @@
             <lightning-layout-item size="12">
                 <h1 class="slds-text-heading_medium slds-p-top-medium">Elevate Gateway Management</h1>
             </lightning-layout-item>
+            <lightning-layout-item size="12" class="slds-text-align_center">
+                <lightning-input 
+                    type="toggle"
+                    data-id="enableGatewayAssignment"  
+                    label={CUSTOM_LABELS.psEnableGatewayAssignment} 
+                    onchange={handleToggle} 
+                    checked={gatewayAssignmentEnabled}
+                    message-toggle-active="Enabled" 
```

Comment:
I think the toggle is supposed to say On / Off when used for a feature or setting (https://confluence.internal.salesforce.com/display/public/DOCTEAM/toggle)

Reply:
@ErinWiedemer If that's the standard, so be it. Production for Advanced Mapping for Data Import & Gift Entry uses "Enabled". So does RD2 Enablement. Since we're using the same look we adopted the same verbiage, but happy to comply with the standard. @screcco-sfdo

Reply:
@npsp-reedestockton I run into this all the time. I have these standards, but nothing we have matches them ;) If it's not a a lot of effort to change, I think we should follow them, but I also think it's probably not a big deal, since as you noted, there's a history of using enabled/disabled.

Reply:
@ErinWiedemer @screcco-sfdo Very easy to change. Presumably our work on future stuff will reflect the standards, so over time we should move toward consistency. Thanks!

Reply:
@ErinWiedemer @npsp-reedestockton updated

Reply:
Thank you!